src/components/CounterBadge/CounterBadge.st.css

```diff
@@ -0,0 +1,70 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, applyOpacity, fallback;
+}
+
+/*Overrides*/
+:vars {
+    /*
+        Background color
+        @default color-5 with opacity 0.1
+    */
+    CounterBadgeBgColor: --overridable;
+    /*
+        Text color
+        @default priority default and light: color-5, priority primary: color-1
+    */
+    CounterBadgeTextColor: --overridable;
+}
+
+:vars {
+    defaultBgColor: color(fallback(value(CounterBadgeBgColor), color(color-8)));
+    defaultTextColor: color(fallback(value(CounterBadgeTextColor), color(color-5)));
+}
+
+:vars {
+    lightBgColor: color(fallback(value(CounterBadgeBgColor), applyOpacity(color(color-8),0.1)));
+    lightTextColor: color(fallback(value(CounterBadgeTextColor), color(color-8)));
+}
+
+:vars {
+    primaryBgColor: color(fallback(value(CounterBadgeBgColor), color(color-8)));
+    primaryTextColor: color(fallback(value(CounterBadgeTextColor), color(color-1)));
+}
+
+.root {
+    -st-states: withIcon;
+    display: inline-flex;
+    line-height: 20px;
+    font-size: 12px;
+    border-radius: 10px;
+    min-width: 20px;
+    justify-content: center;
+}
+
+.overrideStyleParams {
+    background-color: color(value(defaultBgColor));
+    color: color(value(defaultTextColor));
+}
```

Comment:
Let's put it under the title: /* Variants - including style overrides capabilities */

Reply:
The variables values here should be `primaryBgColor`, `primaryTextColor`. Since this is the default value.
I know that you copied it from the Badge component but according to the spec here we only have 2 priorities (primary and secondary).